src/test/java/br/com/pucgo/perguntaai/PerguntaaiBackendApplicationTests.java

```diff
@@ -1,13 +1,27 @@
 package br.com.pucgo.perguntaai;
 
+import br.com.pucgo.perguntaai.models.User;
+import br.com.pucgo.perguntaai.repositories.UserRepository;
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
 @SpringBootTest
 class PerguntaaiBackendApplicationTests {
 
+	UserRepository userRepository;
 	@Test
-	void contextLoads() {
+	void mustResgisterUser() {
```

Comment:
lembre de pesquisar como se escreve teste de endpoints REST com Spring. Olhe este exemplo: https://gabrielpulga.medium.com/a-beginners-guide-to-unit-testing-crud-endpoints-of-a-spring-boot-java-web-service-api-8ae342c9cbcd